Controllers/Backend/Plentymarkets.php

```diff
@@ -406,6 +406,12 @@ public function saveSettingsAction()
 		$Config->setItemNumberSourceKey($this->Request()->ItemNumberSourceKey);
 		$Config->setItemVariationNumberSourceKey($this->Request()->ItemVariationNumberSourceKey);
 		$Config->setItemBundleHeadActionID($this->Request()->ItemBundleHeadActionID == true ? IMPORT_ITEM_BUNDLE_HEAD : IMPORT_ITEM_BUNDLE_HEAD_NO);
+		$Config->setItemShortDescriptionImportActionID($this->Request()->ItemShortDescriptionImportActionID == true ? IMPORT_ITEM_SHORTDESC : IMPORT_ITEM_SHORTDESC_NO);
+		$Config->setItemLongDescriptionImportActionID($this->Request()->ItemShortDescriptionImportActionID == true ? IMPORT_ITEM_LONGDESC : IMPORT_ITEM_LONGDESC_NO);
+		$Config->setItemKeywordsImportActionID($this->Request()->ItemKeywordsImportActionID == true ? IMPORT_ITEM_KEYWORDS : IMPORT_ITEM_KEYWORDS_NO);
+		$Config->setItemNameImportActionID($this->Request()->ItemNameImportActionID);
+		$Config->setItemPriceImportActionID($this->Request()->ItemPriceImportActionID);
+		$Config->setItemFreetextsImportActionID($this->Request()->ItemFreetextsImportActionID == true ? IMPORT_ITEM_FREETEXTS : IMPORT_ITEM_FREETEXTS_NO);
```

Comment:
ItemFreetextsImportActionID ist beim Import von Freitextfelder in setDetails() Methode nicht berücksitigt. Es müsste noch eingebaut werden , wenn der Import von Freitextfelder auch konfigurierbar sein sollte.